lib/publish.js

```diff
@@ -1,17 +1,24 @@
 const fetch = require('node-fetch')
 const colorize = require('../utils/colorize')
 
+const constants = require('./constants')
+
+const {
+  DEFAULT_SPACE_NAME
+} = constants
+
 module.exports = ({
   registryUrl,
   registryEditor,
   registryToken,
+  spaceName = DEFAULT_SPACE_NAME,
   appSlug,
   appVersion,
   appBuildUrl,
   sha256Sum,
   appType
 }, finishCallback) => {
-  fetch(`${registryUrl}/registry/${appSlug}`, {
+  fetch(`${registryUrl}/${spaceName}/registry/${appSlug}`, {
```

Comment:
The `DEFAULT_SPACE_NAME` is just the "empty string space". In your case, you could get rid of this constant and have a simple default `spaceName = ''` with the following logic (for instance):

```js
fetch(`${registryUrl}/${spaceName ? (spaceName + '/') : ''}registry/${appSlug}`)
```